tools/json.rb

```diff
@@ -0,0 +1,11 @@
+require 'json'
+require 'fileutils'
+require_relative '../lib/package'
+
+Dir['../packages/*.rb'].each do |filename|
+  puts filename.to_s
+  pkgName = File.basename filename, '.rb'
+  puts pkgName.to_s
+  @pkg = Package.load_package(filename, pkgName)
+  puts pkg.name.to_s
+end
```

Comment:
What exactly does this do?

Reply:
The aim is to read each package file and print certain variables (name, url, version, etc) into a json file for repology. The various `puts` calls were for debugging.

Reply:
How does this hep with updates?  I can see where it could be used for testing whether the filename and class name match but just printing the package info is for what purpose?  Is this supposed to replace packages.yaml?

Reply:
By the way, we should not be submitting pull requests with debug lines without setting the Draft status.